__tests__/reducers/edits.js

```diff
@@ -148,10 +148,7 @@ describe('edits reducer', () => {
       )
     ).toEqual({ types: { quality: { isFetching: true } } })
   })
-
-  it('handles REFRESH_STATE', () => {
```

Comment:
Removed this one because it's [also done below](https://github.com/cfpb/hmda-platform-ui/pull/983/files#diff-338bf679e07dc9bdcd7eabe7d00aaffeL164).